config/v1/types_cluster_version.go

```diff
@@ -253,7 +253,7 @@ type Update struct {
 	// or potentially malicious sources.
 	//
 	// +optional
-	Force bool `json:"force"`
+	Force bool `json:"force,omitempty"`
```

Comment:
This doesn't make any sense, the default false is not empty but false, in which case you'll always get that. So I'd propose dropping this. It would only make sense with pointer bool. strings changes are good.